src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -23,27 +23,21 @@
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
-/** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
-  private ArrayList<String> messages = new ArrayList<String>();
-
   
-  @Override
-  public void init() {
-    messages.add("Hello, how are you?");
-    messages.add("Hello, what is your name?");
-    messages.add("Hello, what is your age?");
-  }
+  SmallCityService smallCityService;
+
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    String name = "Abakir";
-    
-    response.getWriter().println(convertToJsonUsingGson(messages));
+    // TODO: Return businessList from smallCityService as JSON
   }
-  private String convertToJsonUsingGson(ArrayList<String> temp) {
-    Gson gson = new Gson();
-    String json = gson.toJson(temp);
-    return json;
+
+    @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    // TODO: Create smallCityService instance from User inputted location
+    
+    smallCityService = new SmallCityService(User user);
```

Comment:
What will be the goal here?

Reply:
The goal here is to take the user inputted location from the JSON on the /data page to create the SmallCityService object. Does that make sense?

Reply:
Yes. But if that's the case, is there a reason to not pass the user location directly, rather than the entire user? I'm not suggesting this is wrong, understanding this is still framework code, but I'm trying to get an understanding of the high-level flow.

Reply:
Ah I see what you mean. I think passing in the Map Location makes a lot more sense here. We'll just have to create that mapLocation object in the servlet from the lat and long values returned through the API in javascript. Should be changed now!